packages/pigment-css-react/src/Grid.jsx

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+/* eslint-disable react/jsx-filename-extension */
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import * as React from 'react';
+
+import { gridAtomics } from './baseAtomics';
+
+
+const itemStyle = {
+  width: 'calc(100% * var(--Column-span) / var(--Column-count) - (var(--Column-count) - var(--Column-span)) * var(--Column-gap) / var(--Column-count))',
+  marginLeft: 'calc(100% * var(--Item-offset) / var(--Column-count) + var(--Column-gap) * var(--Item-offset) / var(--Column-count))'
+}
+
+const containerStyle = {
+  gap: 'var(--Row-gap) var(--Column-gap)',
+}
```

Comment:
I would like to add this to a class, but I didn't know how to go about it. How should I do it @brijeshb42?

Reply:
I think this can use `styled`.

Reply:
Implemented 👍🏼